frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Created_From_Add_Options.feature

```diff
@@ -45,100 +45,102 @@ Examples:
 
 @regression, @smoke
 Scenario Outline: Create a pipeline from git workload with resource type "<resource>" : P-02-TC01, P-02-TC06
-   Given user is at "Import from git" form
+   Given user is at Import from git form
    When user type Git Repo url as "<git_url>"
-   And type Name as "<name>" in General section
+   And type Name as "<pipeline_name>" in General section
    And select "<resource>" radio button in Resources section
-   And select "Add Pipeline" checkbox in Pipelines section
-   And click "Create" button on Add page
-   Then user navigates to topology page
-   And created workload is present in topology page
+   And select Add Pipeline checkbox in Pipelines section
+   And click Create button on Add page
+   Then user redirects to Topology page
+   And created workload "<pipeline_name>" is present in topology page
 
 Examples:
-| git_url                                 | pipeline_name    | resource          | name             |
-| https://github.com/sclorg/nodejs-ex.git | nodejs-ex.git-D  | Deployment        | nodejs-ex.git-D  |
-| https://github.com/sclorg/nodejs-ex.git | nodejs-ex.git-DC | Deployment Config | nodejs-ex.git-DC |
+| git_url                                 | pipeline_name    | resource          |
+| https://github.com/sclorg/nodejs-ex.git | nodejs-ex-git-f  | Deployment        |
+| https://github.com/sclorg/nodejs-ex.git | nodejs-ex-git-fc | Deployment Config |
 
 
 @regression
 Scenario Outline: Create a pipeline from git workload with knative resource type  : P-02-TC07
-   Given cluster is installed with knative operator
-   And user is at "Import from git" form
+   Given openshift cluster is installed with knative operator
+   And user is at Import from git form
    When user type Git Repo url as "<git_url>"
    And type Name as "<name>" in General section
    And select "knative" radio button in Resources section
-   And select "Add Pipeline" checkbox in Pipelines section
-   And click "Create" button on Add page
-   Then user navigates to topology page
-   And created workload is present in topology page
+   And select Add Pipeline checkbox in Pipelines section
+   And click Create button on Add page
+   Then user redirects to Topology page
+   And created workload "<workload_name>" is present in topology page
```

Comment:
```suggestion
   And user will see workload "<workload_name>" is present in topology page
```

Reply:
Updated as "user is able to see workload "<pipeline_name>" on topology page"